docs/release-notes/release-notes-0.18.0.md

```diff
@@ -274,6 +274,10 @@ bitcoin peers' feefilter values into account](https://github.com/lightningnetwor
   maintain a healthy connection to the network by checking the number of
   outbound peers if they are below 6.
 
+* [exitNoSuccessOnShutdown](https://github.com/lightningnetwork/lnd/pull/8659)
```

Comment:
`exitNoSuccessOnShutdown` is the branch name... doesnt mean much to the users reading the change log :)

Reply:
Yeah, when I updated the release notes I didn't have text in mind to fit in the 80 chars limit (since URL takes much) so I went with the branch name :D.

Gonna update this!